### ISSUE TITLE
Preferred way of citation

### ISSUE THREAD

**Issue by ThomasTeodorowicz** [qubvel/segmentation_models#89]
Is there a preferred way of citing Segmentation Models in research articles? Currently I would point to the Github repo like this:

~~~~
@misc{Yakubovskiy:2019,
Author = {Pavel Yakubovskiy},
Title = {segmentation models 0.2.0},
Howpublished = {\url{https://github.com/qubvel/segmentation_models}},
Month = {04},
Year = {2019}} 
~~~~

**Comment by qubvel** [comment]
Hi @ThomasTeodorowicz 
I think this way is ok! 
Please let me know about your article, I would be happy to read it :smile:

**Comment by ThomasTeodorowicz** [comment]
All right, thank you for the answer.
It is just my master's thesis, but if you are still interested, I would be happy to send it to you when it is done 😄

**Comment by qubvel** [comment]
Sure, if it is relevant for me. What is the topic of your thesis?

**Comment by ThomasTeodorowicz** [comment]
I'm creating a tool where I use feature extractors to feed a Random Forest with training data to create segmentations from user labels.  The idea now is to try to use features extracted by pre-trained NNs as additional input data. I use your library to obtain feature maps that have the same size as the input image.

**Comment by qubvel** [comment]
With what kind of data are you working?
